src/modules/wholeBodyDynamics/main.cpp

```diff
@@ -360,10 +360,30 @@ class wholeBodyDynamics: public RFModule
         icub_type.head_version = 1;
         icub_type.legs_version = 1;
 
-        if (rf.check("headV2"))
+        // set icub_type.head_version int to 1 (default) or 2
+        if (rf.check("head_version"))
         {
-            yInfo("'headV2' option found. Using icubV2 head kinematics.\n");
-            icub_type.head_version = 2;
+            if (rf.find("head_version").asDouble()==1.0)
+            {
+                yInfo("\"head_version\" %g requested => Using V1 head kinematics.\n",
+                      rf.find("head_version").asDouble());
+            }
+            else if (rf.find("head_version").asDouble()==2.0 ||
```

Comment:
It should be ok, but I never feel very comfortable with this kind comparisons between floating point values... Is this really necessary?